packages/uzbekistan/README.md

```diff
@@ -0,0 +1,49 @@
+# 🇺🇿 Uzbekistan
+
+[![License: CC BY 4.0](https://img.shields.io/badge/License-CC%20BY%204.0-blue.svg)](https://creativecommons.org/licenses/by/4.0/)
+
+## Contents:
+
+### All Regions:
+* Andijon
+* Buxoro
+* Farg‘ona
+* Jizzax
+* Namangan
+* Navoiy
+* Qashqadaryo
+* Samarqand
+* Sirdaryo
+* Surxondaryo
+* Toshkent
+* Xorazm
+
+### Sea area:
+* Orol Dengizi
+
+### Autonomous Republic:
+* Qoraqalpog‘iston
+
+## Installation
+
+### npm
+
+`npm install --save @svg-maps/uzbekistan`
+
+### yarn
+
+`yarn add @svg-maps/uzbekistan`
+
+## Credits
+
+This map is based on the work of [MapSVG](https://mapsvg.com).
+
+The original map is available [here](https://mapsvg.com/maps/japan) under the [Creative Commons Attribution 4.0 International](https://creativecommons.org/licenses/by/4.0/) license.
+
+### Changes
+
+* Remove unnecessary attributes
+* Replace title by name attributes
+* Add viewBox
+* Rename ids
+* Sort `<path/>` alphabetically
```

Comment:
Could you indicate the real changes please?

Reply:
Done